drivers/ethernet/eth_mcux.c

```diff
@@ -471,8 +504,12 @@ static void eth_mcux_phy_event(struct eth_context *context)
 		break;
 	case eth_mcux_phy_state_read_status:
 		/* PHY Basic status is available. */
+#if defined(CONFIG_ETH_MCUX_NO_PHY_SMI) && ETH_MCUX_FIXED_LINK
+		link_up = 1;
```

Comment:
As the variable is pool, do `link_up = true;` instead

Reply:
Ok.